resources/tentative/complex-static-html/.gitignore

```diff
@@ -0,0 +1,3 @@
+/dist/*
```

Comment:
FYI - we check in the dist folder so it can be run directly from a static file server with no build step

Reply:
(and having this included in the PR would make it easier to check locally & get to a decision on the checklist items in the description)

Reply:
Added them. Also added the path to the files I meant on the checklist.